projects/protocompile/project.yaml

```diff
@@ -0,0 +1,10 @@
+homepage: "https://github.com/bufbuild/protocompile"
+primary_contact: "jhumphries@buf.build"
+auto_ccs:
+  - "p.antoine@catenacyber.fr"
```

Comment:
@catenacyber, I carried this over from the previous configuration. But wasn't sure if you still want to be CC'ed on these, especially now that the source of truth repo has changed. Would you like me to remove this line?

Reply:
Thanks @jhump got carrying this.
You can indeed remove me

Reply:
Done!